### ISSUE TITLE
adapt wiki

### ISSUE THREAD

**Issue by liam-harrington** [lae/ansible-role-proxmox#142]
Hey,

i was wondering if it would be possible to change the sample on your wiki,
cause i think the common way to do this is:

- hosts: all
  become: yes
  roles:
    - role: geerlingguy.ntp
        ntp_manage_config: yes
        ntp_servers:
          - clock.sjc.he.net
          - clock.fmt.he.net
          - clock.nyc.he.net
    - role: lae.proxmox
        pve_group: all
        pve_reboot_on_kernel_update: yes

not:

- hosts: all
  become: True
  roles:
    - {
        role: geerlingguy.ntp,
        ntp_manage_config: true,
        ntp_servers: [
          clock.sjc.he.net,
          clock.fmt.he.net,
          clock.nyc.he.net
        ]
      }
    - {
        role: lae.proxmox,
        pve_group: all,
        pve_reboot_on_kernel_update: true
      }

This is problematic, because people might start copying your code...
The missing ',[]{}' will also make it easier to maintain.
Please change your sample.

**Comment by lae** [comment]
I don't believe that was the common way when this role was originally written? Did a recent Ansible version allow for that syntax?

But I also can't see how it's problematic unless Ansible did not retain backwards compatibility, as that exact bit of syntax has been working for years as-is.

**Comment by jmariondev** [comment]
I have been using the "non-braces" syntax for a couple years, I don't think I've ever used the braces. This just seems like a quirk of YAML that it allows defining maps that way.

However, it seems like both are supported. I'm of the opinion that the non-braced format "looks better", but I can see the argument that it's less explicit (indentation in YAML can be confusing).

**Comment by lae** [comment]
I do think the non-braced format looks better/is easier to manage (if we're avoiding vars files entirely). I just never saw it used anywhere back when I was learning Ansible/studying other roles. I think I attempted to use a non-braced format a few times several years ago and just fumbled, probably because of incorrect indentation (and thus have been using it this way for simple playbooks, since).

If it's actually been supported for a long time then I don't mind it getting rewritten.
